### PR TITLE
support Kubernetes 1.30

### PR DESCRIPTION
**Pull request by ushitora-anqou**

### CODE REVIEW

File: internal/runners/pvc_autoresizer.go [hunk 1]
```diff
@@ -98,17 +94,15 @@ func (w *pvcAutoresizer) getStorageClassList(ctx context.Context) (*storagev1.St
 	return &scs, nil
 }
 
-func (w *pvcAutoresizer) reconcile(ctx context.Context) error {
+func (w *pvcAutoresizer) reconcile(ctx context.Context) {
 	scs, err := w.getStorageClassList(ctx)
 	if err != nil {
 		w.log.Error(err, "getStorageClassList failed")
-		return nil
```
Comment (daichimukai) [inline internal/runners/pvc_autoresizer.go:105]: When we fail to get the storage class, we cannot go on. `return` is necessary.

File: internal/runners/pvc_autoresizer.go [hunk 2]
```diff
@@ -98,17 +94,15 @@ func (w *pvcAutoresizer) getStorageClassList(ctx context.Context) (*storagev1.St
...
 	}
 
 	vsMap, err := w.metricsClient.GetMetrics(ctx)
 	if err != nil {
 		w.log.Error(err, "metricsClient.GetMetrics failed")
-		return nil
```
Comment (daichimukai) [inline internal/runners/pvc_autoresizer.go:111]: Ditto.

File: internal/runners/pvc_autoresizer.go [hunk 3]
```diff
@@ -117,7 +111,6 @@ func (w *pvcAutoresizer) reconcile(ctx context.Context) error {
 		if err != nil {
 			metrics.KubernetesClientFailTotal.Increment()
 			w.log.Error(err, "list pvc failed")
-			return nil
```
Comment (daichimukai) [inline internal/runners/pvc_autoresizer.go:120]: Ditto.

**Review by daichimukai — CHANGES_REQUESTED** [review]

File: internal/runners/pvc_autoresizer.go (hunk 1 above)
Comment (ushitora-anqou) [reply internal/runners/pvc_autoresizer.go:105]: Thanks for your feedback. I fixed my code. Could you review this again?

**Review by daichimukai — APPROVED** [review]

**Review by satoru-takeuchi — APPROVED** [review]

**Pull request merged**